app/components/Edit/EditSections.js

```diff
@@ -138,25 +136,23 @@ function postNotes(notesObj, promises, uriObj) {
       if (key < 0) {
         const uri = `/api/${uriObj.path}/${uriObj.id}/notes`;
         promises.push(dataService.post(uri, { note: currentNote }));
+      } else if (currentNote.isRemoved) {
+        const uri = `/api/notes/${key}`;
+        promises.push(dataService.APIDelete(uri));
```

Comment:
I have VSCode auto lint/beautify so it did some refactoring. I looked through the changes and trust it.